tests/lib/tools/journal-state

```diff
@@ -73,7 +79,29 @@ get_journalctl_log_from_cursor(){
     fi
 }
 
-sync_journalctl_log(){
+_sync_journalctl_log(){
     journalctl --flush || true
     journalctl --sync || true
 }
+
+if [ $# -eq 0 ]; then
+    show_help
+    exit 1
+fi
+
+action=
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -h|--help)
+            show_help
+            exit 0
+            ;;
+        *)
+            action=$(echo "$1" | tr '-' '_')
+            shift
+            break
+            ;;
+    esac
+done
+
+"$action" "$@"
```

Comment:
How does this behave on unknown action?

Reply:
Added a check to validate the command is supported.